CFS/0. Overview/服務概觀.md

```diff
@@ -1 +1,31 @@
+---
+tags: Guide, CFS, EN
+title: 服務概觀 | en
+GA: UA-155999456-1
+---
+
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+<style>
+</style>
+
+# <img class="icon" src="https://i.imgur.com/HmCCsr5.png" width="40" height="40"> Service Overview 
+
+## Cloud management and file sharing, providing TWCC services to access data for AI/ML pipeline, data analysis, etc.
+
+
+### Cold and Hot storage
+  - **File upload/download**
+    Provides a connection to TWCC to store files in cloud storage.
+  - **File access and management (list/search/version history/metadata)**
+    Provide file version management and metadata search management.
+  - **File Lifecycle Management - Hot and Cold Storage Data Transfer**
+    A policy set by user to autumatically or manually move in-frequent access file to cold storage for long-term data storage, and files can be migrated between different storages at any time.
+
+### File Sharing
+  - **Set sharing permissions and control file access**
+    Different permissions of members in the project can be set according to needs for file sharing and security management.
```

Comment:
Different permissions of the project members can be set according to needs of file sharing and security management.